examples/dummy_plugin/dummy_plugin/homepage.py

```diff
@@ -0,0 +1,52 @@
+from nautobot.core.apps import HomePageItem, HomePagePanel
+
+from .models import DummyModel
+
+
+def get_dummy_data(request):
+    examples = DummyModel.objects.all()
+    if not examples:
+        examples = (
+            DummyModel.objects.create(name="Example 1", number=100),
+            DummyModel.objects.create(name="Example 2", number=200),
+            DummyModel.objects.create(name="Example 3", number=300),
+        )
+    return examples
+
+
+layout = (
+    HomePagePanel(
+        name="Organization",
+        items=(
+            HomePageItem(
+                name="Dummy Models",
+                model=DummyModel,
+                weight=150,
+                link="plugins:dummy_plugin:dummymodel_list",
+                description="List dummy plugin models.",
+                permissions=["dummy_plugin.view_dummymodel"],
+            ),
+        ),
+    ),
+    HomePagePanel(
+        name="Dummy Plugin",
+        weight=250,
+        items=(
+            HomePageItem(
+                name="Dummy Models",
+                link="plugins:dummy_plugin:dummymodel_list",
+                model=DummyModel,
+                description="List dummy plugin models.",
+                permissions=["dummy_plugin.view_dummymodel"],
+                weight=100,
```

Comment:
As per https://github.com/nautobot/nautobot/pull/674#discussion_r675548478, maybe change one of these two `HomePageItems` to have a custom template of its own, so as to verify and demonstrate that `HomePageItem` custom templates work as intended?

Or do we already have a `HomePageItem` with a custom template somewhere in core that I'm not aware of?